packages/rest/src/__tests__/unit/rest.server/rest.server.unit.ts

```diff
@@ -201,6 +201,25 @@ describe('RestServer', () => {
         const expressApp = server.expressApp;
         expect(expressApp.get('strict routing')).to.equal(true);
       });
+      it('honors openApiSpec.info in config', async () => {
```

Comment:
The title should be `openApiSpec.initialObject` now.

Reply:
ohh, I fix it now